src/selectors/entities/posts.js

```diff
@@ -32,11 +32,19 @@ export function makeGetReactionsForPost() {
         getReactionsForPosts,
         (state, postId) => postId,
         (reactions, postId) => {
-            return Object.values(reactions[postId] || {});
+            return Object.values(reactions[postId] || []);
         }
     );
 }
 
+const getReactionsForPost = makeGetReactionsForPost();
+
+export function getUserIdsFromReactions(state, postId) {
```

Comment:
Since this is taking the postId as an argument, it should be done as a factory function that returns an instance of the selector